recipes/vs2015_runtime/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: vs2015_runtime
+  # version is the update version reported from the registry
+  version: "14.0.25123"
+
+build:
+  number: 0
+  msvc_compiler: 14.0
+  skip: True  # [not win]
+
+test:
+  commands:
+    # only test one of the api dll's.  We test all the others, but there are lots of these.
+    - if not exist Library\bin\api-ms-win-crt-string-l1-1-0.dll exit 1
```

Comment:
First of them passed, hooray!

I have heard a shadowy mention of a planned way to keep [pypi dependencies fresh](https://github.com/conda-forge/invoke-feedstock/pull/2) (or at least notify us when they change). We would want to pay attention to [this](http://sxp.microsoft.com/feeds/msdntn/Visual_Studio_Product_Updates) as well, then... indeed, it looks like Visual Studio Update 3 RC dropped on Tue, 07 Jun 2016 17:42:15 Z, after the first preview was launched on Tue, 10 May 2016 17:18:21 Z, which suggests sometime in July everything might break again :)